README.md

```diff
@@ -19,6 +19,18 @@ CRAM discussions can also be found on the [samtools-devel mailing list][samdev-m
 
 The **[tabix.tex]** and **[CSIv1.tex]** quick references summarize more recent index formats: the tabix tool indexes generic textual genome position-sorted files, while CSI is [htslib]'s successor to the BAI index format.
 
+Unaligned sequence data files
+-----------------------------
+
+We do not define nor endorse any dedicated unaligned sequence data format.
+Instead we recommend storing such data in one of the alignment formats (SAM, BAM or CRAM) with the unmapped flagged set.
+However for completeness, we list the commonest formats below with external links.
+
+[FASTA] is an early sequence-only format originally defined by William Pearson's tool of the same name.
+
+[FASTQ] was designed as a replacement for FASTA, combining the sequence and quality information in the same.
+It has no formal definition and several incompatible variants, but is described in a paper by Cock et al.
```

Comment:
Could you add a year or title (or even better, a DOI) so that it's not so vague?